api/capi/include/nnstreamer.h

```diff
@@ -40,16 +40,16 @@ extern "C" {
 
 /**
  * @brief The virtual name to set the video source of camcorder in Tizen.
- * @details If an application needs to access the camcorder to construct the pipeline, set the virtual name as a video source element.
+ * @details If an application needs to access the camera device to construct the pipeline, set the virtual name as a video source element.
  *          Note that you have to add 'http://tizen.org/privilege/camera' into the manifest of your application.
  * @since_tizen 5.5
  */
 #define ML_TIZEN_CAM_VIDEO_SRC "tizencamvideosrc"
 
 /**
  * @brief The virtual name to set the audio source of camcorder in Tizen.
- * @details If an application needs to access the camcorder to construct the pipeline, set the virtual name as an audio source element.
- *          Note that you have to add 'http://tizen.org/privilege/camera' into the manifest of your application.
+ * @details If an application needs to access the recorder device to construct the pipeline, set the virtual name as an audio source element.
+ *          Note that you have to add 'http://tizen.org/privilege/recorder' into the manifest of your application.
```

Comment:
One question here. The recorder privilege (http://tizen.org/privilege/recorder) describes both video and audio.
But, the comments mention only an audio source. Is it intended?
A recorder device seems to be regarded as an audio device.

Reply:
Well, JeongMo Yang who is in charge of both camera and microphone, gave this information. 
I will ask him again.